redirector/netlify.toml

```diff
@@ -1,4 +1,9 @@
 [[redirects]]
-from = "/*"
```

Comment:
I don't quite understand why this line is being changed / what this line change actually does.

Reply:
Sorry, I thought I put the detailed execution plan somewhere on github but I can't find it now. I added it here https://github.com/dandi/dandi-archive/issues/1030#issuecomment-3070458070.

We want `gui-staging.dandiarchive.org` to redirect to `sandbox.dandiarchive.org` going forward, so that means that the `gui-staging-dandiarchive-org` Netlify app will turn into a simple redirector like we currently do for the `gui-dandiarchive-org` project. So, since both `gui-staging-dandiarchive-org` and `gui-dandiarchive-org` will be deployed from `redirector/netlify.toml`, it's necessary to differentiate the two by specifying full URLs instead of just `/*`